### PR TITLE
Low level patches

### PR DESCRIPTION
**Pull request by brucemiller**
This PR is a collection of smallish fidelity improvements that turned up while exploring the feasibility of reading `latex.ltx` directly.

### CODE REVIEW

File: t/daemon/fatal_100.tex [hunk 1]
```diff
@@ -9,8 +9,8 @@ \section{Last heading here}
 % Any nonsence that generates errors would do,
 % let's just call \the on undefined macros in a loop:
 \makeatletter%
-\def\letters{a}%
-\def\letterloop{\edef\thisname{\csname\letters\endcsname} \the\thisname \g@addto@macro\letters{a} \letterloop}
+\def\letters{Fuu}%
```
Comment (dginev) [inline t/daemon/fatal_100.tex:12]: Funny!

**Review by dginev — APPROVED** [review]
I took a quick, but detailed, read through -- all looks very sensible. 

It's great to see the finer details w.r.t catcode comparisons improve yet again.

**Comment by dginev** [comment]
There is a transient failure in installation step of the Windows CI, which is safe to ignore:
```
Attempt to get headers for http://strawberryperl.com/download/5.26.3.1/strawberry-perl-5.26.3.1-64bit.msi failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'http://strawberryperl.com/download/5.26.3.1/strawberry-perl-5.26.3.1-64bit.msi'. Exception calling "GetResponse" with "0" argument(s): "Unable to connect to the remote server"
```

It is a shame really. But it should disappear if reran.

**Comment by dginev** [comment]
> it should disappear if reran.

It did.

File: lib/LaTeXML/Core/Definition/CharDef.pm [hunk 2]
```diff
@@ -40,9 +43,17 @@ sub setValue {
 
 sub invoke {
   my ($self, $stomach) = @_;
-  my $cs = $$self{internalcs};
-  # Tracing ?
-  return (defined $cs ? $stomach->invokeToken($cs) : undef); }
+  my $value     = $$self{value};
+  my $mathglyph = $$self{mathglyph};
+  # A dilemna: If the \chardef were in a style file, you're prefer to revert to the $cs
```
Comment (dginev) [inline lib/LaTeXML/Core/Definition/CharDef.pm:48]: dilemma*

**Comment by brucemiller** [comment]
Woohoo! `git add --patch` really works! Thanks @tkw1536 !!

**Comment by brucemiller** [comment]
I think I've done everything that makes sense at this stage. But since there are so many subtle low-level changes, an arXiv regression test would be useful.

**Comment by brucemiller** [comment]
OK, seems to cause no major regressions, so lets merge.

**Pull request merged**